docs/source/usage/plugins/radiation.rst

```diff
@@ -324,7 +323,7 @@ Command line flag                        Output description
                                          The spectral intensity is only summed over the last radiation ``dump`` period.
 ``--<species>_radiation.radPerGPU``      Same output as *totalRadiation* but only summed over each GPU. 
                                          Because each GPU specifies a spatial region, the origin of radiation signatures can be distinguished.
-*radiationHDF5*                          In the folder  ``radiationHDF5``, hdf5 files for each radiation dump and species are stored.
+*radiationOpenPMD*                       In the folder  ``radiationOpenPMD``, openPMD files (currently hdf5) for each radiation dump and species are stored.
```

Comment:
Does it always write in .h5 file format? If so, maybe it should be added that not just opnePMD API is required, but one with hdf5 backend

Reply:
For now, it only writes hdf5 files using the openPMD API. 
I will add a comment.